src/openforms/authentication/contrib/digid_eherkenning_oidc/views.py

```diff
@@ -97,6 +124,18 @@ class eHerkenningOIDCAuthenticationCallbackView(
     plugin_identifier = "eherkenning_oidc"
     auth_backend_class = OIDCAuthenticationEHerkenningBackend
 
+    def get_error_message_parameters(
+        self, error: str, error_description: str
+    ) -> tuple[str, str]:
+        if (
+            error == "access_denied"
+            and error_description
+            and error_description == "The user cancelled"
+        ):
+            return ("_eherkenning-message", LOGIN_CANCELLED)
```

Comment:
this could also be eidas instead of eherkenning, see `MESSAGE_PARAMETER` in the eherkenning contrib views :thinking: 

I don't know if those get special treatment in the frontend though.